src/timeseriesflattener/logger.py

```diff
@@ -0,0 +1,83 @@
+"""Exampel of how to setup a logger with sensible defaults."""
+
+import logging
+from pathlib import Path
+
+import coloredlogs
+
+from timeseriesflattener.utils import PROJECT_ROOT
+
+
+def setup_logger(
```

Comment:
This is surprisingly painful..

Reply:
This is also quite elaborate - a pretty good, easy example is:

```
logger = logging.getLogger(name)
coloredlogs.install()
```

And then you've got pretty decent defaults :-)